RestructuredText/syntax_test_restructuredtext.rst

```diff
@@ -60,3 +66,69 @@ This is *italic*.
 
 this is **bold**.
 ..      ^^^^^^^^ markup.bold.restructuredtext
+
+
+Directive tests
+--------------
+
+.. not possible to test a multi-line directive, since comments cannot appear
+.. mid-directive
```

Comment:
Syntax tests prioritize the "comment test marker token" over valid syntax, meaning you can use them anywhere you like and still test the previous line. You do this in line 87 as well.

Reply:
The presence of a comment marker invalidates the directive contents. This RST produces a note

```
.. note::

    My note
```

This produces only a comment

```
.. note::
.. <-- test comment here

    My note
```

Reply:
To write such a test, I would need a marker like
```
.. note ::
    .. <-[2] scope.name
    .. <-[2] means "apply to column 2
```

Reply:
Ah, found a workaround in the rST to get what I wanted

Reply:
The usual "workaround" is to repeat the mutli-line construct and then add tests at a later point.

Reply:
That's exactly what I ended up doing in the first rev of this, but I think my approach now of doing both is better anyway. Removed this comment, so will resolve this review chain.